### PR TITLE
Fix Macedonian display name in ComponentModel CultureInfo converter

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

File: src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs [hunk 1]
```diff
@@ -58,7 +58,7 @@ public override bool CanConvertTo(ITypeDescriptorContext context, Type destinati
         /// </summary>
         public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
         {
-            // Hack, Only when GetCultureName returns culture.Name, we use CultureInfoMapper 
+            // Hack, Only when GetCultureName returns culture.Name, we use CultureInfoMapper
```
Comment (danmosemsft) [inline src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs]: Can we avoid the word hack? Maybe it's just me. More like a special case.

Comment (tarekgh) [reply src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs]: I fixed it.

File: src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs [hunk 2]
```diff
@@ -434,7 +434,7 @@ private static class CultureInfoMapper
                     {"Lower Sorbian (Germany)", "dsb-DE"},
                     {"Luxembourgish (Luxembourg)", "lb-LU"},
                     {"Macedonian", "mk"},
-                    {"Macedonian (Former Yugoslav Republic of Macedonia)", "mk-MK"},
+                    {"Macedonian (North Macedonia)", "mk-MK"},
```
Comment (vivmishra) [inline src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs]: This should be "Republic of North Macedonia"

File: src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs [hunk 3]
```diff
@@ -434,7 +434,7 @@ private static class CultureInfoMapper
                     {"Lower Sorbian (Germany)", "dsb-DE"},
                     {"Luxembourgish (Luxembourg)", "lb-LU"},
                     {"Macedonian", "mk"},
```
Comment (vivmishra) [inline src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs]: This I think needs to change to "North Macedonia" as "mk" is associated with North Macedonia only. 

North Macedonia (ISO country code MK)
.mk, the Internet country code top-level domain for North Macedonia

Comment (tarekgh) [reply src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs]: > This I think needs to change to "North Macedonia" as "mk" is associated with North Macedonia only.

Who said that? mk is neutral and can be associated with any country uses the Macedonian language. That is what we did in the desktop.

File: src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs (hunk 2 above)
Comment (tarekgh) [reply src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs]: From where you get this info? It should be just North Macedonia.

Comment (vivmishra) [reply src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs]: It is optional we can keep it as-is.

File: src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs (hunk 3 above)
Comment (vivmishra) [reply src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs]: Sorry, I read that as Macedonia instead of Macedonian.

**Comment by tarekgh** [comment]
corefx-ci (Packaging All Configurations x64_Debug) failure is unrelated

```
D:\a\1\s\.dotnet\sdk\3.0.100-preview3-010431\NuGet.targets(119,5): error : Failed to retrieve information about 'transport.Microsoft.NETCore.Runtime.CoreCLR' from remote source 'https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/flatcontainer/transport.microsoft.netcore.runtime.coreclr/index.json'. [D:\a\1\s\external\runtime\runtime.depproj]
D:\a\1\s\.dotnet\sdk\3.0.100-preview3-010431\NuGet.targets(119,5): error :   A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. [D:\a\1\s\external\runtime\runtime.depproj]
```

**Pull request merged**